### PR TITLE
Feature/fsr 50

### PR DESCRIPTION
**Pull request by nikiwycherley**
https://eaflood.atlassian.net/browse/FSR-50

polygons on outlook map are able to be selected and tool tip appears creating base for tickets in next sprint to display information

### CODE REVIEW

File: server/src/js/components/map/layers.js [hunk 1]
```diff
@@ -39,18 +39,54 @@ window.flood.maps.layers = {
...
-        key: window.flood.model.bingMaps, // + '&c4w=1&cstl=rd&src=h&st=ar|fc:b5db81_wt|fc:a3ccff_tr|fc:50a964f4;sc:50a964f4_ard|fc:ffffff;sc:ffffff_rd|fc:50fed89d;sc:50eab671;lbc:626a6e_mr|lbc:626a6e_hr|lbc:626a6e_st|fc:ffffff;sc:ffffff_g|lc:dfdfdf_trs|lbc:626a6e',
+        key: window.flood.model.bingMaps,
         imagerySet: 'RoadOnDemand'
       }),
       visible: false,
       zIndex: 0
     })
   },
 
+  // road: () => {
```
Comment (teddmason) [inline server/src/js/components/map/layers.js:66]: Does this need leaving in?

**Review by teddmason — APPROVED** [review]
All looks fine to me, see how it tests :)

**Review by JFSCH — COMMENTED** [review]
Looks fine.

File: server/src/js/components/map/layers.js (hunk 1 above)
Comment (nikiwycherley) [reply server/src/js/components/map/layers.js]: no, was left in for reference but doesnt need to be there, ill remove it

**Review by nikiwycherley — COMMENTED** [review]
removed commented code

**Review by teddmason — APPROVED** [review]

**Pull request merged**